querydsl-core/pom.xml

```diff
@@ -98,4 +98,21 @@
       </plugin>
     </plugins>
```

Comment:
Can you verify that the introduced dependency for `classgraph` is in `provided` scope here? (as it is brought transitively with the dependency on codegen)

Reply:
Did you intend to comment somewhere else? Core doesn't depend on codegen

Reply:
classgraph scope is `test` for all modules except codegen and codegen-utils